### DISCUSSION TITLE
clash verge rev 延迟时间为什么比sing-box 延迟时间多

### DISCUSSION THREAD

**Discussion by jerry-rs** [clash-verge-rev/clash-verge-rev#1677] [Q&A] [answered]
1. 在本地macos电脑安装clash verge rev 和sing-box
2. 采用同一家的机场订阅，导入订阅后，点击测速
3. 发现同样的节点，singbox需要200ms左右，但是clash verge rev 需要300ms~400ms
4. 并且clash verge rev节点时不时的出现timeout,但是singbox 同样的节点有延迟数值

**为什么有如此大的差异，问题应该出现在哪里？**

**Comment by wonfen** [comment]
可以在杂项设置里自定义测试地址。

**Reply by jerry-rs** [reply]
sing-box 和clash verge rev 采用的都是默认地址，那么应该是相同地址

**Reply by ghost** [reply, marked as answer]
> sing-box 和clash verge rev 采用的都是默认地址，那么应该是相同地址

之前研究过这个，有个机场延时特别低，发现配置文件多一个 [unified-delay: true](https://wiki.metacubex.one/en/config/general/#unified-delay)

去拓展配置添加一个，延时就低了
![image](https://github.com/user-attachments/assets/e41269c1-268f-424c-837a-d447cc7a1009)
